### ISSUE TITLE
Updates to current prototype

### ISSUE THREAD

**Issue by yaili** [canonical-websites/vanillaframework.io#11]
- [x] Menu items should be sentence case
- [x] External icon in hero button should be dark grey, and it looks a bit streched
- [x] Accessibility in Guidelines is missing an "i"
- [x] Can we have the normal muted heading in "Who’s using Vanilla"?
- [x] Can the logos pattern be centred in the row?
- [x] Can get involved be h3 style?
- [x] Newsletter button should say Subscribe
- [x] Newsletter button should be to the right of input
- [x] Legal info in the footer should be sentence case

Thanks!

**Comment by yaili** [comment]
There is a double border above the footer
